touchforms/formplayer/experiments.py

```diff
@@ -0,0 +1,102 @@
+import laboratory
+import json
+import logging
+
+diff_logger = logging.getLogger('formplayer_diff')
+timing_logger = logging.getLogger('formplayer_timing')
+
+
+class FormplayerExperiment(laboratory.Experiment):
+    session_id_mapping = {}
+
+    def publish(self, result):
+        # if we're starting a new form, we need to store the mapping between session_ids so we can use later
+        if (self.name == "new-form"):
+            control_session_id = json.loads(result.control.value)["session_id"]
+            candidate_session_id = json.loads(result.observations[0].value)["session_id"]
+            FormplayerExperiment.session_id_mapping[control_session_id] = candidate_session_id
```

Comment:
it looks like FormplayerExperiment is being used by client processes when we post to touchforms, so storing this as a dict on the class won't work because the client can be invoked from different processes which won't share this dict.

i think a better way to store the session id mapping is to use redis. you can set a key in redis to be something like `'touchforms-to-formplayer-session-id-%s' % control_session_id`, set its value to be `candidate_session_id`, and set it to expire in 1 week (the max smsforms session length). then you can just look up `'touchforms-to-formplayer-session-id-%s' % control_session_id` anywhere you need it to get the formplayer session id